.gitignore

```diff
@@ -7,4 +7,5 @@
 /pkg/
 /spec/reports/
 /tmp/
-*.gem
\ No newline at end of file
+*.gem
+/.idea/
```

Comment:
Thanks for the suggestion @kblake. Question: do you foresee any harm leaving this here? It should be a no-op, except for those of us using RubyMine, where it helps.